playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
>    Why do we include: ../openshift-cluster/initialize_groups.yml in
>    byo/openshift-checks and then include: ../openshift-cluster/evaluate_
>    groups.yml in common/openshift-checks?

The first is for translating BYO group names into `g_` names. It's a pretty
simple translation in BYO, more complex for other starting points. Then
these names are fed into the second which further defines `oo_first_master`
and such that are to actually be used in the roles.

>    Why do we have tags: always in include: ../openshift-cluster/evaluate_
>    groups.yml, but not in the evaluate_groups include?

This is probably cruft. Basically nothing actually works if you specify
tags (maybe some very narrow dev use cases).